docs/source/releases/v1_11_3a0.rst

```diff
@@ -16,6 +16,7 @@ Version 1.11.3a0 (2023-08-19)
 * Require sphinx<7.2 due to bugs introduced 2023-08-17 7.2.x release
 * Add .vscode settings repository to "full_install.sh"
 * Add script to enable geoips environment
+* Replace xRITDecompress with pyPublicDecompWT for seviri_hrit reader
```

Comment:
NOTE: Do not merge until all other PRs are merged, and this one is automatically updated to merge to main.

Reply:
Done and ready to go.